docs/document/content/reference/distsql/syntax/ral/global-rule/alter-traffic-rule.cn.md

```diff
@@ -17,10 +17,13 @@ lableName ::=
   identifier
 
 trafficAlgorithmDefination ::=
-  'TRAFFIC_ALGORITHM' '(' 'TYPE' '(' 'NAME' = trafficAlgorithmTypeName (',' 'PROPERTIES' '(' key '=' value (',' key '=' value)* ')')? ')' ')'
+  'TRAFFIC_ALGORITHM' '(' 'TYPE' '(' 'NAME' '=' trafficAlgorithmTypeName (propertiesDefination)? ')' ')'
 
 loadBalancerDefination ::=
-  'LOAD_BALANCER' '(' 'TYPE' '(' 'NAME' = loadBalancerName (',' 'PROPERTIES' '(' key '=' value (',' key '=' value)* ')')? ')' ')'
+  'LOAD_BALANCER' '(' 'TYPE' '(' 'NAME' '=' loadBalancerName (propertiesDefination)? ')' ')'
+
+propertiesDefination ::=
+  ',' 'PROPERTIES' '(' key '=' value (',' key '=' value)* ')'
```

Comment:
Hi @Qianyi951015 ,  I think a definition should not start with a `comma`, do you agree?

Reply:
Change like this better?
```
'LOAD_BALANCER' '(' 'TYPE' '(' 'NAME' '=' loadBalancerName (',' propertiesDefination)? ')' ')'
propertiesDefination ::=
   'PROPERTIES' '(' key '=' value (',' key '=' value)* ')'
```

Reply:
Yes.